theano/tensor/nnet/abstract_conv2d.py

```diff
@@ -0,0 +1,545 @@
+"""
+FIXME
+"""
+
+__docformat__ = "restructuredtext en"
+
+import logging
+
+import theano
+from theano.tensor import (as_tensor_variable, patternbroadcast)
+from theano.tensor import TensorType
+from theano.gof import Apply, Op
+from theano.gof import local_optimizer
+from theano.tensor.opt import register_specialize_device
+
+# Cpu implementation
+from theano.tensor.nnet import conv2d as cpu_conv2d, ConvOp
+from theano.tensor.nnet.ConvGrad3D import convGrad3D
+from theano.tensor.nnet.ConvTransp3D import convTransp3D
+
+
+_logger = logging.getLogger("theano.tensor.nnet.conv2d")
+
+
+def conv2d(inputs,
+           filters,
+           inputs_shape=None,
+           filters_shape=None,
+           border_mode='valid',
+           subsample=(1, 1),
+           filters_flip=True):
```

Comment:
Can you rename the parameters to be consistent with [`theano.nnet.conv.conv2d`](https://github.com/ballasn/Theano/blob/conv2d_interface/theano/tensor/nnet/conv.py#L37-L38)?